AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -537,17 +537,21 @@ extension DefaultVideoClientController: VideoClientController {
             dataContainer = Data(dataAsByteArray)
         } else if JSONSerialization.isValidJSONObject(data) {
             dataContainer = try JSONSerialization.data(withJSONObject: data)
+        } else if let dataAsData = data as? Data {
```

Comment:
Feel it's more appropriate to change `data` from `Any` to `Data` instead in the method signature, maybe a new method to avoid breaking change.

Reply:
Why? The top level API takes Any object from builder and this internal API is where we do the converting to pass to media. If we change the method signature, there's still need a place to do the converting from Any.

Reply:
I mean it's more appropriate have a public api with `Data` type.

Reply:
I think SDK doing the heavy lifting to converting multiple type of data is nice for builder and that's the contract of this API. Also inventing a new API is out of this CR's scope.

Reply:
Discussed offline, I'm OK with fixing the existing api, but let's have a team discussion about if we should have explicit data type instead of `Any` for `data`.